opentargets_pharmgkb/evidence_generation.py

```diff
@@ -1,39 +1,105 @@
 import json
+import multiprocessing
 
 import pandas as pd
+from cmat.consequence_prediction.common.biomart import query_biomart
 
+from opentargets_pharmgkb.ols import get_chebi_iri
+from opentargets_pharmgkb.pandas_utils import none_to_nan, explode_column
 from opentargets_pharmgkb.variant_coordinates import get_coordinates_for_clinical_annotation
 
-pgkb_column_names = [
+
+column_names = [
     'Clinical Annotation ID',
     'Variant/Haplotypes',
     'Gene',
+    'split_gene',
+    'ensembl_gene_id',
+    'Drug(s)',
+    'split_drug',
+    'chebi',
     'Level of Evidence',
     'Phenotype Category',
-    'Drug(s)',
     'Phenotype(s)',
     'Genotype/Allele',
+    'all_genotypes',
     'Annotation Text'
 ]
 
 
-def pipeline(clinical_annot_path, clinical_alleles_path, created_date, output_path):
+def pipeline(clinical_annot_path, clinical_alleles_path, drugs_path, created_date, output_path):
     clinical_annot_table = pd.read_csv(clinical_annot_path, sep='\t')
     clinical_alleles_table = pd.read_csv(clinical_alleles_path, sep='\t')
+    drugs_table = pd.read_csv(drugs_path, sep='\t')
 
     merged_table = pd.merge(clinical_annot_table, clinical_alleles_table, on='Clinical Annotation ID', how='left')
-    rs_only_table = merged_table[merged_table['Variant/Haplotypes'].str.contains('rs')][pgkb_column_names]
-
+    # Restrict to variants with rsIDs
+    rs_only_table = merged_table[merged_table['Variant/Haplotypes'].str.contains('rs')]
```

Comment:
We should count how many variant do not have an `rs`

Reply:
created #10, we can add other things to count as we think of them